client/src/routes/login/+page.svelte

```diff
@@ -67,6 +71,7 @@
 			</div>
 			<button
 				class="mt-6 flex w-full justify-center rounded-md border border-zinc-600 bg-zinc-700 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-zinc-600 focus:outline-none focus:ring-2 focus:ring-zinc-500 focus:ring-offset-2"
+				on:click={continueWithGitHub}
```

Comment:
```suggestion
				onclick={continueWithGitHub}
```
Brug svelte 5 syntax